presto-parquet/src/main/java/com/facebook/presto/parquet/writer/ParquetSchemaConverter.java

```diff
@@ -149,7 +149,7 @@ else if (decimalType.isShort()) {
 
     private org.apache.parquet.schema.Type getMapType(MapType type, String name, List<String> parent)
     {
-        parent = ImmutableList.<String>builder().addAll(parent).add(name).add("map").build();
+        parent = ImmutableList.<String>builder().addAll(parent).add(name).add("key_value").build();
```

Comment:
@ugurmeet this change broke the iceberg test, I tried to change it back, then iceberg test looks ok.  But looks like it would failed hive test, am I right?

Reply:
Ok, I think the root cause is the parquet version in the root pom of presto is still 1.11.0,  and iceberg connector is using parquet version in the root pom somehow.  @ugurmeet could you upgrade the parquet in the root pom of presto to 1.11.1?  then the issue should be fixed.  Thanks!

Reply:
Thanks a lot @beinan , all tests are passing now

Reply:
Thank you so much!  You work unblocks a dozen of new features on hive and iceberg connector!